experimental/cloudstrapper/playbooks/roles/vars/defaults.yaml

```diff
@@ -0,0 +1,29 @@
+
+#------------------
+#Default variables
+#Do not need to be changed
+#Caution before you change
+#------------------
+
+secgroupDefault: secgroupMagmaProto
+bucketDefault: bucket-mantle-canada
+keyBoot: keyMagmaBoot
+keyHost: keyMagmaHost
+
+devOpsCloudstrapper: ec2MagmaDevopsCloudstrapper
+primaryCloudstrapper: ec2MagmaCloudstrapper
+devOpsAmi: imgCloudstrapperBase
+
+userBootstrap: ubuntu
+userMagma: "{{ userBootstrap }}"
+userAgw: admin
```

Comment:
minor nit: userAgw -> userGateway or userGw ?

Reply:
Using Agw to distinguish Access Gateway from any future references to other gateways such as FedGw.